libraries/adaptive-expressions/src/functionUtils.ts

```diff
@@ -593,6 +593,31 @@ export class FunctionUtils {
         return locale;
     }
 
+        /**
+     *
+     * @param args An array of arguments.
+     * @param format A format string.
+     * @param locale A locale string.
+     * @param maxArgsLength The max length of a given function.
+     */
+    public static determineFormatAndLocale(
+        args: unknown[],
+        maxArgsLength: number,
+        format: string,
+        locale = 'en-us'
+    ): { format: string; locale: string } {
+        if (maxArgsLength >= 2) {
+            if (args.length === maxArgsLength) {
+                format = FunctionUtils.timestampFormatter(args[maxArgsLength - 1] as string);
```

Comment:
Might as well just perform the `typeof` type checking rather than `as` casting.

Reply:
changed to typeof

Reply:
Looks like this was only fixed in the above function, `determineLocale`.

Reply:
fixed now